.github/scripts/trymerge.py

```diff
@@ -47,13 +47,33 @@
               name
             }
             oid
-            checkSuites(filterBy: {appId: 12274}, first: 1) {
+            checkSuites(first: 50) {
```

Comment:
The max page size in GitHub GraphQL is 100. I assume the bot has enough API tokens to make that work, and this would make this a little more future-proof